persistence/src/commonMain/db_user/com/wire/kalium/persistence/Messages.sq

```diff
@@ -237,6 +237,9 @@ SELECT * FROM Message ORDER BY Datetime(date) DESC LIMIT ? OFFSET ?;
 selectById:
 SELECT * FROM Message WHERE id = ? AND conversation_id = ?;
 
+selectAllMessagesByConversationId:
```

Comment:
Wouldn't be better to just create a query to remove all messages by conversation id?
```
deleteAllMessagesByConversationId:
DELETE FROM Message WHERE conversation_id = ?;
```
and instead of getting a list of message ids and deleting them one by one, just execute this one query to delete all at once

Reply:
In SQL there also the concept of a cascade delete. Where if you delete a conversation it would also automatically delete all messages rows which has that conversation as it's foreign key.

Reply:
I was doing it in a 2-step process, but ended up going for the ON DELETE CASCADE, simplifying the code.